From a1f4a635e00ad9e0febadf26fbd0ec9bc0ca7ac8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Tue, 29 May 2018 14:22:28 +0200
Subject: [PATCH] busybox: fix sysntpd /sbin/ntpd is installed as link to
 /sbin/ntpd

I don't know the original state but at the moment both ntpd and busybox
implementation are installed to /sbin/ntpd. There is no implementation
in /usr/sbin/ntpd to my knowledge. But either way it's invalid at least
for busybox at the moment and this fixes it.
---
 package/utils/busybox/files/sysntpd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/utils/busybox/files/sysntpd b/package/utils/busybox/files/sysntpd
index e693e40..2b9eb6b 100755
--- a/package/utils/busybox/files/sysntpd
+++ b/package/utils/busybox/files/sysntpd
@@ -4,7 +4,7 @@
 START=98
 
 USE_PROCD=1
-PROG=/usr/sbin/ntpd
+PROG=/sbin/ntpd
 HOTPLUG_SCRIPT=/usr/sbin/ntpd-hotplug
 
 get_dhcp_ntp_servers() {
-- 
2.17.0

